content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigKerberosSCALE.md

```diff
@@ -20,63 +20,37 @@ A [keytab ("key table")](https://web.mit.edu/kerberos/krb5-devel/doc/basic/keyta
 
 TrueNAS SCALE allows users to configure general Kerberos settings, as well as realms and keytabs.
 
-### Kerberos Settings
-
-Users can configure Kerberos settings by navigating to **Directory Services** and clicking **Settings** in the **Kerberos Settings** window.
-
-![KerberosSettingsSCALE](/images/SCALE/KerberosSettingsSCALE.png "Kerberos Settings")
-
-| Field | Description |
-|---------|-------|
-| **Appdefaults Auxiliary Parameters** | Additional Kerberos application settings. See [[appdefaults] in krb.conf(5)](https://web.mit.edu/kerberos/krb5-1.12/doc/admin/conf_files/krb5_conf.html#appdefaults) for settings and usage syntax. |
-| **Libdefaults Auxiliary Parameters** | Additional Kerberos library settings. See [[libdefaults] in krb.conf(5)](https://web.mit.edu/kerberos/krb5-1.12/doc/admin/conf_files/krb5_conf.html#libdefaults) for settings and usage syntax. |
-
-### Kerberos Realms
+## Kerberos Realms
+{{< hint info >}}
+TrueNAS automatically generates a realm after you configure AD or LDAP.
+{{< /hint >}}
 
 Users can configure Kerberos realms by navigating to **Directory Services** and clicking **Add** in the **Kerberos Realms** window.
 
-![KerberosRealmsSCALE](/images/SCALE/KerberosRealmsSCALE.png "Kerberos Realms Form")
+![KerberosRealmsScreen](/images/SCALE/22.12/KerberosRealmsScreen.png "Kerberos Realms Screen")
 
 Enter the realm and key distribution (KDC) names, then define the admin and password servers for the realm.
 
-TrueNAS automatically generates a realm after you configure AD or LDAP.
-
-### Kerberos Keytabs
-
-Kerberos keytabs let you join an Active Directory or LDAP server without a password.
+Click **Save**. 
 
-TrueNAS automatically generates a Keytab after you configure AD or LDAP.
+## Kerberos Keytabs
+{{< hint info >}}
+TrueNAS automatically generates a keytab after you configure AD or LDAP.
+{{< /hint >}}
 
+Kerberos keytabs let you join an Active Directory or LDAP server without a password. 
```

Comment:
This is incorrect. keytabs are generally not used to perform a domain join. A domain join is not the same thing as an LDAP bind. Domain joining is the process whereby an account is created on the remote domain controller for our TrueNAS server and associated DNS and kerberos changes are made to allow access to our server via active directory accounts. No such joining process is used in LDAP.